src/sentry/api/endpoints/organization_details.py

```diff
@@ -173,6 +174,8 @@
         bool,
         GITHUB_COMMENT_BOT_DEFAULT,
     ),
+    ("aggregatedDataConsent", "sentry:aggregated_data_consent", bool, DATA_CONSENT_DEFAULT),
+    ("genAIConsent", "sentry:gen_ai_consent", bool, DATA_CONSENT_DEFAULT),
```

Comment:
should we add these to the [test_various_options](https://github.com/getsentry/sentry/blob/72f6e06d0aff1f2742da5c21102bb569b6a52887/tests/sentry/api/endpoints/test_organization_details.py#L393) test?